src/firebase/firebase.module.ts

```diff
@@ -1,5 +1,5 @@
 import { Module } from '@nestjs/common';
-import { FirebaseAdminModule } from '@tfarras/nestjs-firebase-admin';
+import { FirebaseAdminCoreModule } from '@tfarras/nestjs-firebase-admin';
```

Comment:
Why this change,? It was deprecated right.

Reply:
I resolved some conflicts that had arisen after ur PR.. and did npm install and it started to show correctly

Reply:
Revert this change. 
Delete your node_modules.
then run npm install
`@tferras/firebase ` is using version version 2, where FireAdminCoreModule is not available.

Reply:
Done